src/vm/comthreadpool.cpp

```diff
@@ -678,6 +710,19 @@ void WINAPI BindIoCompletionCallbackStub(DWORD ErrorCode,
 #endif // !FEATURE_PAL
 }
 
+FCIMPL1(VOID, ThreadPoolNative::TraceAllocateNativeOverlapped, LPOVERLAPPED lpOverlapped)
+{
+    FCALL_CONTRACT;
+
+    HELPER_METHOD_FRAME_BEGIN_RET_0();
```

Comment:
This needs to be without `RET`. (It won't build otherwise.)

Reply:
Weird, it built somehow.